packages/shared/src/components/auth/AuthHeader.tsx

```diff
@@ -5,17 +5,23 @@ import { CloseModalFunc } from '../modals/common';
 import { Modal } from '../modals/common/Modal';
 import { ModalHeaderKind } from '../modals/common/types';
 
-export interface AuthModalHeaderProps {
+export interface AuthHeaderProps {
+  simplified: boolean;
   title: string;
   className?: string;
   onBack?: CloseModalFunc;
 }
 
-function AuthModalHeader({
+function AuthHeader({
+  simplified = false,
   title,
   className,
   onBack,
-}: AuthModalHeaderProps): ReactElement {
+}: AuthHeaderProps): ReactElement {
+  if (simplified) {
+    return <h2 className="font-bold text-center typo-title2">{title}</h2>;
+  }
```

Comment:
Basically simplified means outside of the modal, so header needs to be inlined to show per screen :)